packages/manager/apps/hpc-vmware-managed-vcd/src/utils/veeamBackupId.ts

```diff
@@ -0,0 +1,9 @@
+import IVcdOrganization from '@/types/vcd-organization.interface';
+
+const getOrganizationUuid = (organization?: IVcdOrganization) =>
+  organization?.id.split(
+    `${organization?.currentState.region.toLowerCase()}-`,
+  )[1];
+
+export const getBackupIdFromOrganization = (organization: IVcdOrganization) =>
+  `${getOrganizationUuid(organization)}-veeam-backup`;
```

Comment:
Has `getOrganizationUuid` accept undefined.

It's volunteers we not accept undefined ?

It's not important the result will be : `-veeam-backup` ?

Reply:
It will be in the shared module with veeam/vcd and is tested there